### PR TITLE
Add FreeDNS client installation for Raspbian

### PR DESCRIPTION
**Pull request by Pant**
This is an installation script for FreeDNS (http://freedns.afraid.org/) client for Raspbian. The client is basically a Bash script. It detects the difference in the registered IP on the record with the current IP address of the Raspbian board. If there is a difference, it runs a wget command and updates the record with the current IP address.

### CODE REVIEW

File: etc/nextcloudpi-config.d/freedns.sh [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#
+#
+#
+
+ACTIVE_=yes
+UPDATEURL_=https://freedns.afraid.org/dynamic/update.php?
+UPDATEHASH_=Rjh1aXdKQlUxWmFCd1lHT2JsNEpiQm55OjE1ODg0ODM4
+DOMAIN_=nextcloud.ignorelist.com
+UPDATEINTERVAL_=30
+DESCRIPTION="FreeDNS client"
```
[the remark this answers is not in the document]
Comment (Pant) [reply etc/nextcloudpi-config.d/freedns.sh]: +1 -> "DDNS FreeDNS client (need account)"

[the remark this answers is not in the document]
Comment (nachoparker) [reply etc/nextcloudpi-config.d/freedns.sh]: reminder: change hash ;P

File: etc/nextcloudpi-config.d/freedns.sh [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+           --backtitle "NextCloudPi configuration" \
+           --title --title "Instructions for FreeDNS client 
+Set the time in seconds in UPDATEINTERVAL. 
+>>> Long interval may lead to not updating your IP address for long time. <<<" \
+  20 90
+}
+
+install()
+{
+	apt-get install dnsutils -y
```
[the remark this answers is not in the document]
Comment (Pant) [reply etc/nextcloudpi-config.d/freedns.sh]: +1

File: etc/nextcloudpi-config.d/freedns.sh [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+ACTIVE_=yes
+UPDATEURL_=https://freedns.afraid.org/dynamic/update.php?
+UPDATEHASH_=Rjh1aXdKQlUxWmFCd1lHT2JsNEpiQm55OjE1ODg0ODM4
+DOMAIN_=nextcloud.ignorelist.com
+UPDATEINTERVAL_=30
+DESCRIPTION="FreeDNS client"
+URL=${UPDATEURL_}${UPDATEHASH_}
+
+show_info()
```
[the remark this answers is not in the document]
Comment (Pant) [reply etc/nextcloudpi-config.d/freedns.sh]: I would suggest to leave it there until we explain somewhere else what users should have in mind...

**Pull request closed without merging**